packages/common-all/src/SchemaCreationUtils.ts

```diff
@@ -0,0 +1,73 @@
+import YAML from "js-yaml";
+import _ from "lodash";
+import { SchemaData, SchemaTemplate } from "./types";
+
```

Comment:
in common-all, all file names are small case. can we stick to the same convention?

this file could just be called `schema` and we can put schema utils inside of it